src/pages/cart/index.ts

```diff
@@ -0,0 +1,187 @@
+        promocodeInput.id = 'promocode';
+        promocodeInput.className = 'promocode-input';
+        promocodeInput.placeholder = 'Enter promocode';
+
+        const cartSum = document.createElement('div');
+        cartSum.className = 'cart-sum';
+        const cartSumTitle = document.createElement('p');
+        cartSumTitle.className = 'sum-title';
+        cartSumTitle.innerText = 'Summary';
+        const productsSumInfo = document.createElement('div');
+        productsSumInfo.className = 'products-sum-info';
+        const sumProducts = document.createElement('span');
+        const totalPrice = document.createElement('span');
+        const saleInfo = document.createElement('div');
+        saleInfo.className = 'sale-info';
+        const saleText = document.createElement('span');
+        saleText.innerText = 'Sale';
+        const saleAmount = document.createElement('span');
+        saleAmount.innerText = `${sale}$`;
+        const paymentInfo = document.createElement('div');
+        paymentInfo.className = 'payment-info';
+        const paymentText = document.createElement('span');
+        paymentText.innerText = 'Payable';
+        const paymentAmount = document.createElement('span');
+
+        const catBg = document.createElement('img');
+        catBg.src = 'https://i.imgur.com/F8sJeSU.png';
+        catBg.className = 'cart-cat-bg';
+
+        cartContainer.appendChild(cartTitle);
+        cartContainer.appendChild(cartSection);
+        cartSection.appendChild(this.createCartList(currentOrderProducts));
+        cartSection.appendChild(cartCalculations);
+        cartCalculations.appendChild(promocode);
+        promocode.appendChild(promocodeTitle);
+        promocode.appendChild(promocodeInput);
+        cartCalculations.appendChild(cartSum);
+        cartSum.appendChild(cartSumTitle);
+        cartSum.appendChild(productsSumInfo);
+        productsSumInfo.appendChild(sumProducts);
+        productsSumInfo.appendChild(totalPrice);
+        cartSum.appendChild(saleInfo);
+        saleInfo.appendChild(saleText);
+        saleInfo.appendChild(saleAmount);
+        cartSum.appendChild(paymentInfo);
+        paymentInfo.appendChild(paymentText);
+        paymentInfo.appendChild(paymentAmount);
+        cartSum.appendChild(createButton({ buttonText: 'Buy now', onClick: createModal }));
+        const updateCartStats = () => {
+            const currentOrderProducts = CartStore.get();
+            if (!currentOrderProducts.length) {
+                cartSection.style.display = 'none';
+                cartTitle.innerText = 'Cart is Empty. Go to Catalog';
+                return;
+            }
+
+            const currentTotalPrice = currentOrderProducts.reduce((acc, rec) => acc + rec.amount * rec.price, 0);
+            cartTitle.innerText = `Cart ${CartStore.getQuantity()} items`;
+            totalPrice.innerText = `${currentTotalPrice}$`;
+            sumProducts.innerText = `Products (${CartStore.getQuantity()})`;
```

Comment:
You use `CartStore.getQuantity()` twice, you can use const for it:
`const quantity = CartStore.getQuantity()` and then use `quantity`